wiki/Kiai_time/id.md

```diff
@@ -0,0 +1,21 @@
+stub: true
+tags:
+   - timing section
+   - kiai mode
+   - waktu
+---
+
+# Waktu Kiai
+*Untuk peraturan seputar waktu kiai, lihat: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+**Kiai Time** (atau *kiai*) adalah istilah yang diberikan untuk bagian waktu "khusus" untuk membantu menekankan bagian dari [beatmap](/wiki/Beatmap). Pemain dapat mengetahui apakah bagian dari beatmap memiliki bagian waktu kiai jika mereka melihat efek visual tambahan di latar belakang.
+
+Saat kiai aktif, semua [objek](/wiki/Hit_object) yang dipukul akan berkedip mengikuti [irama ketukan utama](/wiki/Beatmap_Editor/Timing) dan bintang akan meledak dari kursor saat [hit circle](/wiki/Hit_object/Hit_circle) dipukul. Jika tidak, mereka akan jatuh dari kursor saat [slider](/wiki/Hit_object/Slider) atau [spinner](/Wiki/Hit_object/Spinner) diikuti.
+
+Beatmapping
+Waktu kiai paling sering digunakan untuk "bagian terkuat dari sebuah lagu" yang biasanya adalah bagian chorus. Karena bagian kiai adalah bagian terkuat dari sebuah lagu, mereka biasanya sedikit lebih menantang daripada bagian [beatmap](/wiki/Beatmap) lainnya untuk mengikuti lagu tersebut.
```

Comment:
```suggestion
Waktu kiai paling sering digunakan untuk "bagian terkuat dari sebuah lagu", yang mana biasanya adalah bagian-bagian dari chorus. Dikarenakan bagian kiai adalah bagian terkuat dari sebuah lagu, biasanya sedikit lebih menantang daripada bagian dari sisa-sisa [beatmap](/wiki/Beatmap) tanpa penggunaan waktu kiai.
```

Reply:
bit confused to understand to localization indo. translation didn't follow the understandable spelling. hope this useful.

Reply:
ok, applied